.rubocop.yml

```diff
@@ -1,10 +1,23 @@
-AllCops:
+require:
+  - rubocop-rspec
+
+Metrics/BlockLength:
   Exclude:
-    - "spec/**/*"
+    - microformats.gemspec
+    - spec/**/*.rb
 
 Metrics/LineLength:
   Enabled: false
 
+RSpec/ExampleLength:
+  Max: 15
+
+RSpec/MultipleExpectations:
+  Max: 10
+
+RSpec/NestedGroups:
+  Max: 6
+
```

Comment:
We should revisit these settings down the line, but for now, this should cut down on some of the less useful warnings from rubocop-rspec.